src/components/OcSelect.vue

```diff
@@ -1,10 +1,19 @@
 <template>
-  <vue-select ref="select" :value="model" class="oc-select" v-bind="$attrs" v-on="$listeners">
-    <template v-for="(index, name) in $scopedSlots" #[name]="data">
-      <slot :name="name" v-bind="data"></slot>
-    </template>
-    <div slot="no-options" v-translate>No options available.</div>
-  </vue-select>
+  <div>
+    <label :for="this.$attrs['input-id']" v-text="label" />
```

Comment:
Where is this supposed to come from? How is this wired up with vue-select?

Reply:
Ah I assume vue-select has this prop? In other places we seem to use an id prop, maybe we should stick to that

Reply:
https://vue-select.org/api/props.html#inputid

Reply:
https://github.com/owncloud/owncloud-design-system/blob/17e7e6fd968a536f56f91db8d30a0317535583b3/src/components/OcTextInput.vue#L70